README.md

```diff
@@ -110,6 +118,9 @@ $ brew install cmake \
   leveldb \
   ossp-uuid \
   boost \ openssl
+
+# If building with GPS support
+$ brew install gps
```

Comment:
Hmm... gps doesn't show up as a package for me in brew.  
```
brew search gps
bgpstream         gpsbabel          gpsd ✔            gpsim             osm-gps-map
Caskroom/cask/gpsdump
```

Am I overlooking something?